code/game/machinery/cryopod.dm

```diff
@@ -587,6 +587,8 @@
 				if(Gh.client && Gh.client.holder) //just in case someone has a byond name with @ at the start, which I don't think is even possible but whatever
 					to_chat(Gh, "<span style='color: #800080;font-weight: bold;font-size:4;'>Warning: Your body has entered cryostorage.</span>")
 	log_admin("<span class='notice'>[key_name(E)] entered a stasis pod.</span>")
+	if(SSticker.mode.name == "nuclear emergency" || SSticker.mode.name == "wizard")
+		SSblackbox.record_feedback("tally", "TDM_quitouts", 1, "TDM Cryopods")
```

Comment:
This is optional, but this would let us sort by gamemode also to my knowledge. Will need someone with database knowledge to confirm
```suggestion
		SSblackbox.record_feedback("nested tally", "TDM_quitouts", 1, list("TDM Cryopods", SSticker.mode.name))
```